docs/network/interfaces_and_networks.md

```diff
@@ -91,61 +92,55 @@ definition.
 apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
-  name: bridge-test
+  name: linux-bridge-net-ipam
 spec:
   config: '{
       "cniVersion": "0.3.1",
-      "name": "bridge-test",
-      "type": "bridge",
-      "bridge": "br1",
+      "name": "mynet",
       "disableContainerInterface": true
+      "plugins": [
+        {
+          "type": "bridge",
+          "bridge": "br10",
+          "ipam": {
+            "type": "whereabouts",
+            "range": "10.1.1.0/24"
+          },
+          "macspoofchk": true
+        }
+      ]
     }'
 ```
 
 With following definition, the VMI will be connected to the default pod
-network and to the secondary Open vSwitch network.
+network and to the secondary bridge network, referencing the `NetworkAttachmentDefinition` 
+shown above(in the same namespace)
 
 ```yaml
 kind: VM
 spec:
   domain:
     devices:
       interfaces:
-        - name: default
-          masquerade: {}
-          bootOrder: 1   # attempt to boot from an external tftp server
-          dhcpOptions:
-            bootFileName: default_image.bin
-            tftpServerName: tftp.example.com
-        - name: ovs-net
-          bridge: {}
-          bootOrder: 2   # if first attempt failed, try to PXE-boot from this L2 networks
+      - masquerade: {}
+        name: default
+      - bridge: {}
+        name: linux-bridge-with-ipam
   networks:
   - name: default
     pod: {} # Stock pod network
-  - name: ovs-net
+  - name: linux-bridge-with-ipam
     multus: # Secondary multus network
-      networkName: ovs-vlan-100
+      networkName: linux-bridge-net-ipam #ref to NAD name
 ```
-
+#### Multus as primary network provider
 It is also possible to define a multus network as the default pod
-network with [Multus](https://github.com/intel/multus-cni). A version of
-multus after this [Pull
-Request](https://github.com/intel/multus-cni/pull/174) is required
-(currently master).
-
-**Note the following:**
-
--   A multus default network and a pod network type are mutually
-    exclusive.
-
--   The virt-launcher pod that starts the VMI will **not** have the pod
-    network configured.
+network by indicating `spec.networks.multus.default=true`.
+See [Multus](https://github.com/intel/multus-cni) documentation for further information<br>
+**Note:** that a multus `default` network and a `pod` network type are mutually
+exclusive.
```

Comment:
The note formal misses starting `>` character on each of the note lines.
Without it, the note will not have a markdown note panel.
```suggestion
> *Note:** that a multus `default` network and a `pod` network type are mutually
> exclusive.
```

Reply:
done